roles/openshift_facts/library/openshift_facts.py

```diff
@@ -640,25 +645,25 @@ def get_local_facts_from_file(filename):
 
 
 class OpenShiftFactsUnsupportedRoleError(Exception):
-    """OpenShift Facts Unsupported Role Error"""
+    """Facts Unsupported Role Error"""
```

Comment:
@ashcrow :+1: on Origin Facts...  probably a bit of a rabbit hole of a change since openshift_facts is a dependency or included in quite a few places.